app/Policies/AlbumPolicy.php

```diff
@@ -183,24 +231,6 @@ public function canEdit(User $user, ?AbstractAlbum $album): bool
 			($album instanceof BaseAlbum && $this->isOwner($user, $album));
```

Comment:
```suggestion
			($album instanceof BaseAlbum && $album->owner->id === $user->id);
```

I would actually try to throw away the method `PhotoPolicy::isOwner` or at least make it not public. `isOwner` is not a right, hence it should not be provided publicly by this class. On top it is such a trivial method that it barely makes sense to have `isOwner` as an internal helper method.

Reply:
made it private.